src/nimble.nim

```diff
@@ -97,7 +97,7 @@ proc processFreeDependenciesSAT(rootPkgInfo: PackageInfo, options: Options): Has
 
   result = solveLocalPackages(rootPkgInfo, pkgList, solvedPkgs)
   if solvedPkgs.len > 0: 
-    displaySatisfiedMsg(solvedPkgs, pkgsToInstall)
+    # displaySatisfiedMsg(solvedPkgs, pkgsToInstall)
```

Comment:
we can simply switch this to debug level where it does make sense to log "positive" progress

Reply:
Will review and rework if necessary the logging system in another pr (test may start to fail if we change how overall logging works)